Annotation/constituent_golden_set.json

```diff
@@ -1490,7 +1699,32 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how she\u2019s thinking and feeling",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "focuses on",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "constituent",
+                "clause": "what she\u2019s going to do",
+                "embedded clauses": []
+            }
+            ,
+            {
+                "predicate": [
+                    {
+                        "str": "focuses on",

```

Comment:
Maybe predicate should be "less so on" (since the "so" replaces the verb "focuses")?